modal/sandbox.py

```diff
@@ -396,7 +397,13 @@ async def _get_task_id(self):
                 await asyncio.sleep(0.5)
         return self._task_id
 
-    async def exec(self, *cmds: str, pty_info: Optional[api_pb2.PTYInfo] = None):
+    async def exec(
+        self,
+        *cmds: str,
+        pty_info: Optional[api_pb2.PTYInfo] = None,
```

Comment:
We might have talked about this before and it might be a TODO, but I don't think we should expose this proto type directly to users. I guess it's mostly for internal use with modal shell anyway - maybe we should document it as internal and underscore it or something for the time being?

Reply:
Added this as an underscored option but left the other so we can smooth migrate on our side. I'll follow up with a removal once we migrate our code.